middlewares/validateArticle.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @description this function checks that fields are not empty
+ * @param {object} req
+ * @param {object} res
+ * @param {function} next
+ * @returns {object} a status code and json object if theres error
+ *  or continue to the next middleware using next()
+ */
+const validateArticle = (req, res, next) => {
+  let { title, description, body } = req.body.article;
```

Comment:
Good job, when I tested the endpoint with an `article` object without `title, description, or body` a failure response was returned. However, when I sent an empty object as in `{ }` it broke. That's because you did not validate for a request body without an article object, please handle that too.

Also, this validation does not ensure the right datatype is passed to specific fields. For example, a title must be a string and not a boolean or integer. This will definitely break the app, so please cater for it, thanks

Reply:
Yes I have tested it, and it has been fixed, well done